pkg/providers/instancetype/zz_generated.bandwidth.go

```diff
@@ -42,6 +42,7 @@ var (
 		"t4g.medium":        256,
 		"m7a.medium":        390,
 		"m7i-flex.large":    390,
+		"r7a.medium":        390,
```

Comment:
I don't think that these changes should be included here. Can you do a `go checkout main <path>` on this to revert it? If we need to update these generated files, we can do so in a separate PR

Reply:
reverted.